src/System.Text.Json/tests/Serialization/ExceptionTests.cs

```diff
@@ -34,14 +34,14 @@ public static void ThrownFromSerializerFails()
         {
             try
             {
-                JsonSerializer.Parse<IDictionary<string, int>>(@"{""Key"":1, ""Key"":2}");
-                Assert.True(false, "Expected JsonException was not thrown.");
+                JsonSerializer.Parse<IDictionary<string, string>>(@"{""Key"":1, ""Key"":2}");
```

Comment:
Why did we end up changing the test case here to string, string? If duplicate keys are supported, then the input string doesn't make sense.

If the goal is now to throw because of a type mismatch, we should be clear in the test input and maybe even the test name, that is what we are testing.